ethereum/eip712/eip712.go

```diff
@@ -63,27 +63,11 @@ func WrapTxToTypedData(
 		Salt:              "0",
 	}
 
-	msgTypes, err := extractMsgTypes(cdc, "MsgValue", msg)
+	msgTypes, err := ExtractMsgTypes(cdc, "MsgValue", msg, feeDelegation != nil)
 	if err != nil {
 		return apitypes.TypedData{}, err
 	}
 
-	if feeDelegation != nil {
-		feeInfo, ok := txData["fee"].(map[string]interface{})
-		if !ok {
-			return apitypes.TypedData{}, sdkerrors.Wrap(sdkerrors.ErrInvalidType, "cannot parse fee from tx data")
-		}
-
-		feeInfo["feePayer"] = feeDelegation.FeePayer.String()
```

Comment:
where is this fee info now?

Reply:
it seems useless, `feeInfo` and `txData` are not returned anywhere

Reply:
@hanchon ^^